### DISCUSSION TITLE
Bimodal distribution using Laguerre tessellation

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#705] [Q&A] [answered]
Dear Romain,
First thnak you so much for this really useful software and really helps me a lot!

My question is, for example, I want to generate a polycrystal having large (mean diameq = 240, std = 12, lognormal distribution) and small grains (mean diameq = 60, std = 3, lognormal distribution) at the same time, the command that I used is here, but to really get the statistics that I want (I mean for example, the mean and std of the large and small grains to really achieve the value that I set) it needs to take a really long time even using supercomputer(in the meanwhile I also need enough number of grains, let's say not less than 20000 grains per realization). Is there anyway to accelarate the speed or could I use any other commands to generate the same thing that I want? Another simple question is that could we control the volume fraction of the sample? I know the value before the "lognormal(......)" means the ratio of number of grains, so we need to calculate volume fraction based on number of grains, or is there any other way to control this?

Command:
```
 neper -T -n from_morpho -domain "cube(2000,2000,2000)" -morpho "diameq:lognormal(240,12)+10*lognormal(60,3),1-sphericity:lognormal(0.1,0.03)" -id 1 -group "diameq>160?1:2" -morphooptiini "weight:radeq,coo:packing"  -morphooptistop val=0.3 -o test
```

Thank you before!
Damu

**Comment by rquey** [comment, marked as answer]
You are already using `-morphooptiini "weight:radeq,coo:packing"`, which is a good point. However, note that it will somewhat struggle to find the initial solution with `weight:radeq`. You can try `weight:0.8*radeq` instead.

It isn't possible to specify volume fractions instead of number fraction yet.

**Reply by damulaoshi** [reply]
Thank you so much for your reply. I will try this command!
